### PR TITLE
Improve API docs based on Best practices

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

Related to https://github.com/ballerina-platform/ballerina-standard-library/issues/3857

## Checklist
- [x] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Checked native-image compatibility

### CODE REVIEW

File: ballerina/Module.md [hunk 1]
```diff
@@ -158,6 +163,8 @@ defined by the `sql:Client` will be supported by the `mysql:Client` as well.
 Once all the database operations are performed, you can close the client you have created by invoking the `close()`
 operation. This will close the corresponding connection pool if it is not shared by any other clients.
 
+**Note**: The client must be closed only at the end of the application lifetime (or closed for graceful stops in a service).
```
Comment (niveathika) [inline ballerina/Module.md:166]:
```suggestion
>**Note**: The client must be closed only at the end of the application lifetime (or closed for graceful stops in a service).
```

File: ballerina/client.bal [hunk 2]
```diff
@@ -22,7 +22,8 @@ import ballerina/sql;
 public isolated client class Client {
     *sql:Client;
 
-    # Initializes the MySQL Client.
+    # Initializes the MySQL Client. It should be kept open throughout the entirety of the application
```
Comment (niveathika) [inline ballerina/client.bal:25]: Lets update this to latest

**Review by niveathika — CHANGES_REQUESTED** [review]

**Comment by niveathika** [comment]
Do the same changes in the master branch too.

**Comment by kalaiyarasiganeshalingam** [comment]
> Do the same changes in the master branch too.

Updated in https://github.com/ballerina-platform/module-ballerinax-mysql/pull/777

**Review by niveathika — APPROVED** [review]

**Pull request merged**